google/resource_dataproc_cluster_test.go

```diff
@@ -648,6 +674,66 @@ resource "google_dataproc_cluster" "basic" {
 `, rnd)
 }
 
+func testAccDataprocCluster_basicWithInternalIpOnlyTrue(rnd string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "dataproc_network" {
+	name = "dataproc-internalip-network"
```

Comment:
Can you also add randomness to this and the subnetwork? This helps out with our automated testing so that if the test fails before the resource is deleted, it doesn't collide with the existing one when trying to create it the next time. Feel free to reuse the same rnd var

Reply:
Fixed in 853a9ee!